docs/dnspolicy/dns-health-checks.md

```diff
@@ -45,6 +45,43 @@ EOF
 ```
 This configuration sets up a DNS health check by creating DNSHealthCheckProbes for the specified `prod-web` Gateway endpoints.
 
+### `additionalHeadersRef`
+
+The `additionalHeadersRef` field specifies a `Secret` used for storing supplementary HTTP headers. These headers are included when sending probe requests and can contain critical information like authentication tokens.
```

Comment:
might be worth mentioning that the secret must be in the same namespace as the DNS Policy

Reply:
good point, will mention